demo/vercel.json

```diff
@@ -0,0 +1,7 @@
+{
+  "functions": {
+    "api/**/*.rs": {
+      "runtime": "ecklf-tmp-runtime-test@1.0.83"
```

Comment:
make sure this is changed later.

Reply:
I'll be switching this out for a canary publish in a bit